commons/src/main/scala/com/lynbrookrobotics/potassium/commons/drivetrain/UnicycleDrive.scala

```diff
@@ -8,55 +8,84 @@ import squants.motion.AngularVelocity
 import squants.{Acceleration, Dimensionless, Length, Percent, Velocity}
 import GenericValue._
 
+trait UnicycleProperties {
+  val maxForwardVelocity: Velocity
+  val maxTurnVelocity: AngularVelocity
+
+  val forwardControlGains: PIDFProperUnitsConfig[Velocity, Acceleration, Length, Dimensionless]
+  val turnControlGains: PIDFConfig[AngularVelocity,
+                                   GenericValue[AngularVelocity],
```

Comment:
The derivative of angular speed is angular acceleration, and its integral is angle. Why not just use these units instead?